app/validators/idv/form_state_id_validator.rb

```diff
@@ -9,6 +9,16 @@ module FormStateIdValidator
                 :state_id_jurisdiction,
                 :state_id_number,
                 presence: true
+
+      validates_with UspsInPersonProofing::TransliterableValidator,
+                     fields: [:first_name, :last_name],
+                     reject_chars: /[^A-Za-z\-' ]/,
+                     message: (proc do |invalid_chars|
+                                 I18n.t(
+                                   'in_person_proofing.form.state_id.errors.unsupported_chars',
+                                   char_list: invalid_chars.join(', '),
+                                 )
+                               end)
```

Comment:
I think if we switch this to "stabby lambda" syntax we may not need the extra wrapping parens anymore, so I just went and pushed that in 5b20738db523c6c8e384a007af693630116a03d1